include/wx/charts/wxdoughnutchart.h

```diff
@@ -29,17 +29,14 @@
 #include "wxdoughnutchartoptions.h"
 
 /// Data for the wxDoughnutChartCtrl control.
-class wxDoughnutChartData
+class wxDoughnutChartData : public wxChartObservableValue<wxVector<wxChartSliceData>>
 {
 public:
     wxDoughnutChartData();
 
     void AppendSlice(const wxChartSliceData &slice);
-
-    const wxVector<wxChartSliceData>& GetSlices() const;
```

Comment:
Let's not remove this function. It's equivalent to GetValue() but it's making it harder for the users since GetValue() is pretty generic as a name and one function is on the base class and the other (AppendSlice) is on the derived class.
Also especially true since the value is actually a vector and GetValue() tends to imply there is only a single item.